service/dap/server.go

```diff
@@ -623,10 +626,55 @@ func (s *Server) onPauseRequest(request *dap.PauseRequest) { // TODO V0
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
-// onStackTraceRequest sends a not-yet-implemented error response.
+// onStackTraceRequest handles ‘stackTrace’ requests.
 // This is a mandatory request to support.
-func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) { // TODO V0
-	s.sendNotYetImplementedErrorResponse(request.Request)
+func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) {
+	// TODO(polina): support depth and cfg via launch/attach args
+
+	goroutineID := request.Arguments.ThreadId
+	// For feature parity with vscode-go, get depth (length of the returned list of Stackframes)
+	// from 'stackTraceDepth' in launch/attach arguments or default to 50.
+	// https://github.com/golang/vscode-go/blob/b4a484b59c4e3fe0966ea1056ddaeb5b77a83da0/src/debugAdapter/goDebug.ts#L373
+	// TODO(polina): Figure out why vscode-go does not specify DAP 'startFrame' + 'levels' value instead,
+	depth := 50
+	// Never set by vscode-go.
```

Comment:
I'm not sure we should have so many mentions of vscode-go here. First, this server will be used for more than just VSCode. Second, vscode-go's DA will go away at some point (in favor of a thin adapter or just plain Delve) and these comments will hang.

Instead, I recommend opening an issue about vscode-go parity, and just referring to the issue here if more context is needed in comments